template.rb

```diff
@@ -45,14 +45,23 @@ def apply_template!(template_root)
   apply 'config/template.rb'
   apply '.gitignore.rb'
 
+  if ENV['CI']
+    @add_devise = true
+    apply '.template/addons/devise/template.rb'
+  elsif yes?("Would you like to add Devise gem?")
+    @add_devise = true
+    apply '.template/addons/devise/template.rb'
+  end
+
   after_bundle do
     use_source_path template_root
 
+    generate 'devise:install' if @add_devise
```

Comment:
can we move this one to `devise` addon itself?

Reply:
Moved to` devise/template.rb` https://github.com/nimblehq/rails-templates/pull/250/commits/069359d26a95bb75d8b1aadab4cea6553423a6f2

Reply:
🤔  it's still here @noor-ahmed something wrong?